### ISSUE TITLE
Filtering empty array items that sit between non-empty values

### ISSUE THREAD

**Issue by mrmartineau** [PaulSavignano/filter-empty#1]
Given an array like this: 
```js
["One", {}, {}, {}, {}, "Two", {}];
```
I would expect `filter-empty` to return this

```js
["One", "Two"];
```

but it instead returns this

```js
[
  "One",
  null,
  null,
  null,
  null,
  "Two"
]
```

Do you think it would be hard to remove the in-between empty values?

Here is a fork of your CodeSandbox showing the behaviour: https://codesandbox.io/s/filter-empty-forked-cfn6n?file=/src/index.js

**Comment by PaulSavignano** [comment]
Hi @mrmartineau, thank you for reporting the issue.  This should be a quick fix.  I'll check into it and circle back with you.

**Comment by mrmartineau** [comment]
That's awesome, thanks mate. Let me know if you need a hand testing it. 👍

**Comment by PaulSavignano** [comment]
Hi @mrmartineau, fix is published in v0.1.11.  Let me know if you find any issues and thanks again for your reporting!

**Comment by mrmartineau** [comment]
Ah brilliant! Thank you very much. I'll give it a try tomorrow and let you know if there are any issues 👍
